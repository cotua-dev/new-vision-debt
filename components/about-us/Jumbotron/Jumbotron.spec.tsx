import { render, RenderResult } from '@testing-library/react';
import { Jumbotron } from './index';

describe('Jumbotron Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Jumbotron/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
