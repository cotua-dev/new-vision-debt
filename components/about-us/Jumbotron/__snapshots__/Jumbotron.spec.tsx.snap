// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron Component renders 1`] = `
<section
  class="about-us"
>
  <div
    class="about-us-content"
  >
    <h1
      class="title"
    >
      About Us
    </h1>
    <p
      class="content"
    >
      Debt Advisors of America is headquartered in San Diego, California. Debt Advisors of America is a member of the American Fair Credit Council; the International Association of Professional Debt Arbitrators; and the Better Business Bureau, with which Debt Advisors of America maintains an A+ rating.
    </p>
    <p
      class="content"
    >
      Debt Advisors of America team members have, together, decades of experience in the debt-restructure industry, and can match you with a partner that will suit your needs.
    </p>
    <p
      class="content"
    >
      We are committed to conducting our business honestly, ethically, and professionally.
    </p>
  </div>
  <picture>
    <source />
    <img
      alt="About Us Jumbotron Image"
      class="w-full h-fit"
      loading="lazy"
    />
  </picture>
</section>
`;
